force-app/main/default/objects/Public_Event__c/fields/Name__c.field-meta.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@t-gul @bmtemakina instead of the custom name field, you can use the standard name. The display format of the object should not be "auto number"